libs/product/src/drivers/magento/transforms/product-transformer.service.ts

```diff
@@ -27,7 +27,7 @@ export class DaffMagentoProductTransformerService {
           return {
             url: response.storeConfig.secure_base_media_url + 'catalog/product' + image.file,
             label: image.label,
-            id: image.id
+            id: image.id.toString()
```

Comment:
Why does this need to be typecast?

Reply:
Because the id from magento is a number, and the daffodil one is a string. The magento object that was typed as `any` is now typed as 
```
{
   label: string,
   file: string,
   position: number,
   disabled: boolean,
   id: number
}
```